pkg/util/testing/wrappers.go

```diff
@@ -189,6 +189,22 @@ func (w *WorkloadWrapper) Admitted(a bool) *WorkloadWrapper {
 	return w
 }
 
+func (w *WorkloadWrapper) WaitingForPodsReady(condStatus bool) *WorkloadWrapper {
```

Comment:
The name is misleading , ` WaitiForPodsReady` is a totally different functionality.

Reply:
Thanks